content/verification/address-validation/release-notes/2024-05-15.md

```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of the Address Validation API. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removing support for US Congressional District. 
+     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing (FIPS) standards.
```

Comment:
```suggestion
     - Adding country code support for both International Standards Organization (ISO) and Federal Information Processing Standards (FIPS). 
```

Reply:
@dsteeve I believe this is the Federal Information Processing Standards, so it should all be capitalized?